lib/tasks/after_tasks.rake

```diff
@@ -517,5 +517,26 @@ namespace :After do
 
     puts "Finished reindexing tags on hidden and unrevealed works"
   end
+
+  desc "Convert user kudos from users with the official role to guest kudos"
+  task(convert_official_kudos: :environment) do
+    official_role = Role.find_by(name: "official")
+    official_users = RolesUser.where(role_id: official_role.id).map(&:user) if official_role.present?
```

Comment:
You can use `official_role.users`.

Reply:
I was going to say "TIL!" but then I realized what you were saying and wow, last night me was a dumbass.